domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -4101,8 +4107,16 @@ public void startPostgresTester() {
   }
 
   /**
-   * Stop Postgres Tester.
-   * Does nothing, if postgresTester is not enabled.
+   * Stop the PostgresTester instance.
+   *
+   * <p>Does nothing, if postgresTester is not enabled or stopPostgresTester() has already been called.
```

Comment:
This line does not seem to match code 100% accurately. Do we even need this line?

Reply:
postgresTester == null if postgresTester is not enabled or stopPostgresTester() has already been called.

Reply:
> postgresTester == null if postgresTester is not enabled or stopPostgresTester() has already been called.

Technically, `stopPostgresTester()` does not make `postgresTester == null` immediately.